test/config/mobile.wdio.local.conf.js

```diff
@@ -161,7 +161,7 @@ exports.config = {
         tagsInTitle: false,
 
         // <number> timeout for step definitions
-        timeout: 20000,
+        timeout: 200000,
```

Comment:
the new value is pretty high for a step to take place, it can make the suite really slow
can we set a midterm value i.e.: 40000 ?